plugins/data_migration/frontend/public/javascripts/countly.views.js

```diff
@@ -1,929 +1,394 @@
-                    }
-                    else {
-                        $("#send_export_button").addClass("disabled");
-                    }
-                }
-            });
-
-            self.create_export_tab();
-
-            //appends import drawer
-
-            self.import_drawer = CountlyHelpers.createDrawer({
-                id: "import-widget-drawer",
-                template: self.import_drawer_template,
-                title: jQuery.i18n.map["data-migration.import-title"],
-                applyChangeTriggers: true,
-                resetForm: function() {
-                    $("#import-widget-drawer .details .section").css('display', 'block');
-                    $("#import-widget-drawer .details .buttons").css('display', 'block');
-                    $("#import-via-token").css('display', 'none');
-                    $("#create_new_token").css('display', 'none');
-                    $("#data_migration_generated_token").css('display', 'none');
-                    $('#import_data_button').addClass('disabled');
-                    $('#import_data_button').css('display', 'block');
-
-                    $('#target-server-data').css('display', 'block');
-                    $('#data-import-type-selector').find(".check[data-from=import-upload]").addClass("selected");
-                    $('#data-import-type-selector').find(".check[data-from=import-token]").removeClass("selected");
-
-                    $('#data-migration-import-via-file').removeClass('file-selected');
-                    $('.dz-filechosen').html('');
-                    if (typeof $("#migration_upload_fallback") !== 'undefined') {
-                        $("#migration_upload_fallback").replaceWith($("#migration_upload_fallback").val('').clone(true));
-                    }
-                    $('#import_data_button').addClass('disabled');
-                    if ($('.fallback').length === 0) {
-                        self.myDropzone.removeAllFiles(); self.myDropzone.enable();
-                    }
-                },
-                onClose: function(callback) {
-                    if ($('#data_migration_generated_token').hasClass('newTokenIsGenerated')) {
-                        CountlyHelpers.confirm(jQuery.i18n.map["data-migration.close-without-copy"], "popStyleGreen", function(result) {
-                            if (result) {
-                                callback(true); //allow closing
+/*global countlyVue, CV, countlyCommon, countlyAuth, countlyGlobal, countlyDataMigration, app */
+(function() {
+    var FEATURE_NAME = 'data_migration';
+
+    var ImportsTab = countlyVue.views.create({
+        template: CV.T("/data_migration/templates/imports-tab.html"),
+        props: {},
+        data: function() {
+            return {
+                list: [],
+                hasCreateRight: countlyAuth.validateCreate(FEATURE_NAME),
+                hasDeleteRight: countlyAuth.validateDelete(FEATURE_NAME)
```

Comment:
We have a mixin for this: `countlyVue.mixins.auth(FEATURE_NAME)` :slightly_smiling_face: